R/DataTypes.R

```diff
@@ -77,6 +78,22 @@ odbcDataType.default <- function(info, obj, ...) {
 }
 
 #' @export
+`odbcDataType.Hive` <- function(info, obj, ...) {
+  switch_type(obj,
+    factor = "VARCHAR(255)",
```

Comment:
If character is "STRING" I think factor and list should be as well, as those types are converted to character vectors on the R side before being passed to the ODBC code.